app/src/end2endTest/java/gov/va/vro/end2end/util/TestSetup.java

```diff
@@ -126,7 +130,7 @@ public String getContentDispositionFilename() {
    * @throws Exception any error to fail the test
    */
   public static TestSetup getInstance(String resourceDir) throws Exception {
-    TestSetup result = new TestSetup();
+    TestSetup result = new TestSetup(resourceDir);
```

Comment:
Use the `resourceDir` as the name of the `TestSetup`. This name is used as the prefix for the PDF file that is optionally saved to disk.